packages/cascara/src/modules/DataModule.module.scss

```diff
@@ -16,16 +16,16 @@ $module-background: #f0f0f0;
   font-weight: 600;
 }
 
+.LabelText {
+  @extend %ModuleLabel;
+}
+
 %ModuleContainer {
   width: 100%;
   padding: $module-padding;
 
   // border-bottom: 1px solid red;
 
-  .LabelText {
-    @extend %ModuleLabel;
-  }
-
```

Comment:
The reason this style is having to be changed is because we are breaking the DOM structure of the modules. The fix is maintaining the same structure. We do not want to be changing the CSS to account for our module markdown changing.

There are other styles that rely on the specific structure of modules being the same that are going to be broken by this change.

Any time we see a need to change CSS that is almost surely a red flag for us that we need to go back and look at our structure.